client/src/components/Main/Main.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from "react";
-import { Nav, Profile, SpotifyLoginBtn } from "../";
+import { Nav, Profile, MatchPage } from "../";
 import { Route, Switch } from "react-router-dom";
+// import MatchPage from "../MatchPage/MatchPage";
+import MatchingCard from "../MatchingCard/MatchingCard";
```

Comment:
Remember to import `MatchingCard` in line 2, instead of separately. MatchingCard is a component exported /components/index.js, just like the other components in that line.
So line should become `import { Nav, Profile, MatchPage, MatchingCard } from "../";` and you should delete line 5.